_distro_map.yml

```diff
@@ -187,6 +187,19 @@ openshift-rosa:
     rosa-preview:
       name: ''
       dir: rosa-preview/
+openshift-rosa-hcp:
+  name: Red Hat OpenShift Service on AWS
+  author: OpenShift Documentation Project <openshift-docs@redhat.com>
+  site: commercial
+  site_name: Documentation
+  site_url: https://docs.openshift.com/
+  branches:
+    enterprise-4.15:
+      name: ''
+      dir: rosa-hcp/
```

Comment:
If we change the ROSA classic dir, then we could make this `rosa` instead of `rosa-hcp`